modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/linktoolbar/plugin.js

```diff
@@ -29,15 +29,15 @@
 
 			const editor = instance.editor;
 
-			instance._toolbar = new CKEDITOR.ui.balloonToolbar(editor);
+			const toolbar = new CKEDITOR.ui.balloonToolbar(editor);
```

Comment:
Hey @markocikos, We are using `instance._toolbar` or `this._toolbar` everywhere in this file. It doesn't breaks anything?

everywhere(sorry, I didn't pull this PR yet, so, I'm just taking a look in GH):

<img width="358" alt="Screen Shot 2021-07-26 at 12 21 15" src="https://user-images.githubusercontent.com/7663513/127014859-35963932-48c9-4286-99eb-4d49bccf227a.png">

<img width="363" alt="Screen Shot 2021-07-26 at 12 21 30" src="https://user-images.githubusercontent.com/7663513/127014906-730d3bf5-93c7-41c2-96fd-40176ccaee92.png">

Reply:
The idea was to be able to toggle both toolbar's visibility, so that when editing a link you'd be able to hide the initial "link toolbar" - since everything has changed, can't really say how this works but I trust you have it covered

Reply:
@diegonvs  This is just a minor SF, not to have a lot repeating `instance._toolbar`s. We are setting `instance._toolbar = toolbar;` at the [end of the function](https://github.com/liferay-frontend/liferay-portal/pull/1284/files/41d4638e89c70f21113af48d8737c104ad838acd#diff-d3f3f4a7755c8388b2659377456a1b8621ae0d8d179ebf544e5a346a0dda7e33R107).